components/_lists.scss

```diff
@@ -0,0 +1,22 @@
+.wds-list {
+  display: block;
+  list-style-type: none;
+  padding: 0;
+  margin: 18px;
+
+  .wds-small-list-item {
```

Comment:
this should be divided into list item and a modifier

.wds-list-item

and
.wds-is-small/big

we should identify what is more commonly used and make it a default
so you don't have to set the modifier always